frontend/scss/components/organisms/consent.scss

```diff
@@ -15,7 +15,8 @@
 @import 'components/atoms/_button.scss';
 
 .#{organism('consent')} {
-  left: 20px;
+  right: 20px;
+  left: auto;
```

Comment:
`auto` is the [default value](https://developer.mozilla.org/de/docs/Web/CSS/Left) for the left-property. Therefore doesn't need to be set explicitly.

Reply:
Left: auto is still in there because there is a left-property in /extensions/amp-consent/0.1/amp-consent.css which has to be overwritten

Reply:
🤦‍♂️ then of course it's fine.